From 511046376c50abe75c938408b22f81af5e55d927 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Thu, 13 Mar 2025 16:21:50 +0100
Subject: [PATCH 1614/1695] dts: stm32: Update LPTIM4 clock source to LSI for
 stm32mp257f-dk

In standby1 the PLL4 is stopped. For the stm32Cube M0+ demo, we need to
use the LPTIM4 when the platform is in Standby1 .

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c3d4e7107d39f22c1e06a7683d5a0523871cfdca dts: stm32: Update LPTIM4 clock source to LSI for stm32mp257f-dk]
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: Ie08108778f59e2938a22d028d882cf40342e245b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/444103
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
index c246ad101..8e64ea319 100644
--- a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
@@ -75,7 +75,7 @@
 		FLEXGEN_CFG(38, XBAR_SRC_PLL5, 0, 3)
 		FLEXGEN_CFG(39, XBAR_SRC_MSI_KER, 0, 0)
 		FLEXGEN_CFG(40, XBAR_SRC_LSE, 0, 0)
-		FLEXGEN_CFG(41, XBAR_SRC_PLL4, 0, 11)
+		FLEXGEN_CFG(41, XBAR_SRC_LSI, 0, 0)
 		FLEXGEN_CFG(42, XBAR_SRC_PLL7, 0, 16)
 		FLEXGEN_CFG(43, XBAR_SRC_PLL4, 0, 23)
 		FLEXGEN_CFG(44, XBAR_SRC_PLL4, 0, 5)
-- 
2.47.2

